_data/software/vim.yml

```diff
@@ -0,0 +1,32 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: Vim is a greatly improved version of the good old UNIX editor Vi.
+    fr: Vim est une version grandement améliorée du bon vieil éditeur UNIX Vi.
+category: '43232100'
+homepageURL:
+  en: 'https://www.vim.org/'
+  fr: 'https://www.vim.org/'
+licences:
+  - URL:
+      en: 'http://vimdoc.sourceforge.net/htmldoc/uganda.html#license'
+      fr: 'http://vimdoc.sourceforge.net/htmldoc/uganda.html#license'
+    spdxID: GPL-2.0-or-later
```

Comment:
VIM has it's own SPDX license value of `VIM`. Should be available now that we're pulling that data
Did see this ttps://github.com/vim/vim/blob/master/LICENSE, but SPDX does point to that sourceforge URL too